<ui version="4.0" >
 <class>UserActionEdit</class>
 <widget class="QWidget" name="UserActionEdit" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QWidget" native="1" name="widget" >
     <layout class="QGridLayout" >
      <property name="leftMargin" >
       <number>1</number>
      </property>
      <property name="topMargin" >
       <number>1</number>
      </property>
      <property name="rightMargin" >
       <number>1</number>
      </property>
      <property name="bottomMargin" >
       <number>1</number>
      </property>
      <item row="0" column="0" >
       <layout class="QVBoxLayout" />
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" >
    <layout class="QVBoxLayout" >
     <item>
      <widget class="QPushButton" name="upButton" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Expanding" hsizetype="Maximum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize" >
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text" >
        <string>↑</string>
       </property>
       <property name="autoRepeat" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="downButton" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Expanding" hsizetype="Maximum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize" >
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text" >
        <string>↓</string>
       </property>
       <property name="autoRepeat" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QPushButton" name="addButton" >
       <property name="text" >
        <string>追加</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="editButton" >
       <property name="text" >
        <string>編集</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeButton" >
       <property name="text" >
        <string>削除</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
